AsyncDisplayKit/ASTableView.mm

```diff
@@ -702,16 +730,35 @@ - (BOOL)shouldBatchFetch
   }
 }
 
-- (void)handleBatchFetchScrollingToOffset:(CGPoint)targetOffset
+- (void)_scheduleCheckForBatchFetching
 {
-  ASDisplayNodeAssert(_batchContext != nil, @"Batch context should exist");
+  // Push this to the next runloop to be sure the scroll view has the right content size
+  dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
This is the case I was referring to, where multiple calls will schedule multiple calls to the Block, Meaning they won't be actually coalesced.